src/goto-harness/function_call_harness_generator.cpp

```diff
@@ -220,6 +274,32 @@ void function_call_harness_generatort::validate_options()
     throw invalid_command_line_argument_exceptiont{
       "required parameter entry function not set",
       "--" FUNCTION_HARNESS_GENERATOR_FUNCTION_OPT};
+  if(
+    p_impl->recursive_initialization_config.min_dynamic_array_size >
+    p_impl->recursive_initialization_config.max_dynamic_array_size)
+  {
+    throw invalid_command_line_argument_exceptiont{
+      "min dynamic array size cannot be greater than max dynamic array size",
+      "--" FUNCTION_HARNESS_GENERATOR_MIN_ARRAY_SIZE_OPT
+      " --" FUNCTION_HARNESS_GENERATOR_MAX_ARRAY_SIZE_OPT};
+  }
+}
+
+std::size_t function_call_harness_generatort::require_one_size_value(
+  const std::string &option,
+  const std::list<std::string> &values)
+{
+  static auto const string_value = require_exactly_one_value(option, values);
```

Comment:
🚫 argument-dependent `static`?! What's going on here?

Reply:
Yep, that's definitely not intended

Reply:
good catch, don't know how we did this.